manager/assets/modext/widgets/modx.panel.welcome.js

```diff
@@ -308,23 +313,13 @@ Ext.extend(MODx.window.DashboardWidgetAdd, MODx.Window, {
                     }, this);
                 }
             }
-        }, {
-            hideLabel: true,
-            xtype: 'displayfield',
-            html: _('widget_add_desc'),
-            anchor: '100%'
         }, {
             fieldLabel: _('widget_size'),
             id: this.ident + '-size',
             xtype: 'modx-combo-dashboard-widget-size',
             name: 'size',
             value: 'half',
             anchor: '100%'
-        }, {
-            hideLabel: true,
-            xtype: 'displayfield',
-            html: _('widget_size_desc'),
```

Comment:
If it is unused need to delete the string from lexicon to avoid a mess.

Reply:
@Alroniks it is still needed for the more detailed view in modx.panel.dashboard.widget.js

Reply:
Good to know! ty @devlanda